ceph_deploy/hosts/centos/install.py

```diff
@@ -161,13 +141,13 @@ def mirror_install(distro, repo_url, gpg_url, adjust_repos, extra_installs=True,
 
         distro.conn.remote_module.write_yum_repo(ceph_repo_content)
         # set the right priority
-        install_yum_priorities(distro)
+        distro.packager.install_priorities_plugin()
         distro.conn.remote_module.set_repo_priority(['Ceph', 'Ceph-noarch', 'ceph-source'])
         distro.conn.logger.warning('alter.d ceph.repo priorities to contain: priority=1')
 
 
-    if extra_installs:
-        pkg_managers.yum(distro.conn, packages)
+    if extra_installs and len(packages):
```

Comment:
why `len(packages)`? lists evaluate to `False` when empty and `True` when not, so you could do `if extra_installs and pacckages:`